polkit: Fix typo

ptomato opened the pull request:
Chalk it up to "don't make people write config files in real programming
languages"

https://phabricator.endlessm.com/T28441

ptomato committed:
polkit: Fix typo

Chalk it up to "don't make people write config files in real programming
languages"

https://phabricator.endlessm.com/T28441

wjt commented:
I would guess (without looking) the build is failing because there is a stale `T28441-debian` branch kicking around somewhere.

andrunko commented:
> I would guess (without looking) the build is failing because there is a stale `T28441-debian` branch kicking around somewhere.

Not really, just checked, the build is using debian-master - not sure what is going on.

andrunko commented:
bot, retest this please

ptomato committed:
tests: Remove expiration date from test key

It doesn't seem to be necessary for this key to have an expiration date.

https://phabricator.endlessm.com/T28441
